clusters.py

```diff
@@ -16,21 +16,18 @@
 import datetime
 
 import gspread
-import google.auth as auth # authenticatiing to google
+import google.auth as auth # authenticating to google
 
 #Global variables/parameters
 
-default_cluster_dir = ('/content/drive/MyDrive/Stanford Audiology Models/'
-                       'Colab Notebooks/')
+default_cluster_dir = ('/content/drive/MyDrive/STAR_Audiology_Model/')
 
 """
 A change is required in the spreadsheet_path. Open the spreadsheet mfb raw audio
 and copy the link from the address bar. The gid code in the url is unique for
 each of the user.
 """
-spreadsheet_path_v1 = ('https://docs.google.com/spreadsheets/d/'
-                       '119_-qrfzGXwV1YBUJdBzvtAQTZnl-xwN7hD9FK5SWfU/'
-                       'edit#gid=84023254')
+spreadsheet_path_v1 = ('https://docs.google.com/spreadsheets/d/1M3pM45Qb2z2pZ_liVssCpOwCVb8fHtaeu1kkiomzxWw/edit#gid=0')
```

Comment:
Did this change?  It's unfortunate that the path is not readable.  :-(

Reply:
yeah, I tried with a blank GSheet. We can ignore this change